### ISSUE TITLE
How to align RibbonTabItem inside Backstage?

### ISSUE THREAD

**Issue by mspd** [fluentribbon/Fluent.Ribbon#896]
Hi@all
I want a similar Backstage like e.g. Word. Some of the RibbonTabItem on top, and some at the bottom. Somehow it doesn't work with Grid. I can't align them at the bottom. Left is what I want, and on the right is what I have at the moment:
![Backstage](https://user-images.githubusercontent.com/78118375/106067023-167a3700-60fe-11eb-819f-f931dcd13418.png)

---
### Environment

- Fluent.Ribbon __v8.0.3
- Windows 10
- .NET Framework 4.7

**Comment by pschimmel** [comment]
You could add two BackstageTab controls like:

```xml
<fluent:Ribbon.Menu>
  <fluent:Backstage>
    <Grid>
     <fluent:BackstageTabControl>
        <fluent:BackstageTabItem Header="New" />
        <fluent:BackstageTabItem Header="Print" />
      </fluent:BackstageTabControl>
      <fluent:BackstageTabControl  VerticalAlignment="Bottom" IsBackButtonVisible="False">
        <fluent:BackstageTabItem Header="Settings" />
        <fluent:Button Header="Close" />
      </fluent:BackstageTabControl>
    </Grid>
  </fluent:Backstage>
</fluent:Ribbon.Menu>
```
This gives you:
![Backstage](https://user-images.githubusercontent.com/9028723/106324304-2ff3be00-6279-11eb-8e27-2859993704ec.png)

It works fine as long as you only have buttons in the lower BackstageTabControl. If you also have BackstageTabItems in there, the lower BackstageTabControl will also maintain it's own selected tab. Maybe you can work around that in code behind.

**Comment by pschimmel** [comment]
While looking at the screenshots, it would be nice to have a built-in way to left-align the header of BackstageTabItems and Buttons in the backstage when I don't intent to use icons for the items.

**Comment by mspd** [comment]
Oh this is a very nice idea. thx!
________________________________
Von: Patrick Schimmel <notifications@github.com>
Gesendet: Freitag, 29. Januar 2021 21:37
An: fluentribbon/Fluent.Ribbon <Fluent.Ribbon@noreply.github.com>
Cc: FR-Technology Support <support@fr-technology.ch>; Author <author@noreply.github.com>
Betreff: Re: [fluentribbon/Fluent.Ribbon] How to align RibbonTabItem inside Backstage? (#896)


You could add two BackstageTab controls like:

<fluent:Ribbon.Menu>
  <fluent:Backstage>
    <Grid>
     <fluent:BackstageTabControl>
        <fluent:BackstageTabItem Header="New" />
        <fluent:BackstageTabItem Header="Print" />
      </fluent:BackstageTabControl>
      <fluent:BackstageTabControl  VerticalAlignment="Bottom" IsBackButtonVisible="False">
        <fluent:BackstageTabItem Header="Settings" />
        <fluent:Button Header="Close" />
      </fluent:BackstageTabControl>
    </Grid>
  </fluent:Backstage>
</fluent:Ribbon.Menu>

This gives you:
[Backstage]<https://user-images.githubusercontent.com/9028723/106324304-2ff3be00-6279-11eb-8e27-2859993704ec.png>

It works fine as long as you only have buttons in the lower BackstageTabControl. If you also have BackstageTabItems in there, the lower BackstageTabControl will also maintain it's own selected tab. Maybe you can work around that in code behind.

**Comment by batzen** [comment]
Fixed this in 124954b1dc4ccdd6736885b0afa0c736a6fb31e5
Details on how to achieve this are in the changelog.
